tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/JavaScript.cs

```diff
@@ -140,6 +142,11 @@ public class JavaScript : LanguageBase
 
             var projectDirectory = Path.Combine(WorkingDirectory, project);
 
+            if(profile){
+                var stripPackageName = primaryPackage.Split(new char[] { '/' })[1]; // "@azure/storage-blob" -> "storage-blob"
+                var profileOutputPath = Path.GetFullPath(Path.Combine(Util.GetProfileDirectory(WorkingDirectory),stripPackageName, $"{packageVersions[primaryPackage]}_{testName}_{iteration}.cpuprofile"));
```

Comment:
```suggestion
                var profileOutputPath = Path.GetFullPath(Path.Combine(Util.GetProfileDirectory(WorkingDirectory), $"{packageVersions[primaryPackage]}_{testName}_{iteration}.cpuprofile"));
```

I would put all the profiles in a flat directory, rather than putting in a subdirectory based on the primary package name, because you might be interested in all the loaded packages, not just the primary one.  If you do this, you don't need to compute `stripPackageName` either.

Reply:
I don't think I understand fully.
Profiles generated for the storage-blob tests will be stored inside the "storage-blob" subfolder.

> because you might be interested in all the loaded packages, not just the primary one

the same profile would contain the core-http and whichever dependency stack storage-blob(the primary package) depends on

Reply:
I don't think creating a storage-blob subfolder is necessary, the test name is enough.  However I'm fine keeping the subfolder if you want.